MUI/src/main/java/mui/MUISettings.java

```diff
@@ -1,14 +1,13 @@
 package mui;
 
-import java.util.ArrayList;
-import java.util.Map;
+import java.util.*;
 
 public class MUISettings {
 
-	public static Map<String, Map<String, Map<String, Object>[]>> SETTINGS =
+	public static Map<String, TreeMap<String, Map<String, Object>[]>> SETTINGS =
 		Map.of(
 			"NATIVE_RUN_SETTINGS",
-			Map.of(
+			new TreeMap(Map.of(
```

Comment:
I thought it was weird behaviour to have the order Manticore options always be randomised every time you re-open the plugin, so I'm using an ordered map (which mimics Python's dict) just had them sorted by ASCII so at least it's consistent.

this is also why `{mcore_binary}` and `{state_server_port}` are surrounded by braces - so that they will appear at the bottom of the options screen outside of alphanumeric rank